packages/selleo-design-core/src/Testimonials.tsx

```diff
@@ -0,0 +1,49 @@
+import { Avatar } from './Avatars';
+
+import { h } from 'preact';
+import cx from 'classnames';
+
+type TestimonialProps = {
+  variant: 'regular' | 'big' | 'vertical';
+};
+
+const avatar = (
+  <img
+    src="https://ui-avatars.com/api/?name=John+Doe"
+    alt="avatar"
+    class="rounded-full w-[100%]"
+  />
+);
+
+export function Testimonial({ variant }: TestimonialProps) {
+  const containerClasses = cx('flex', {
+    'h-[111px] max-h-[111px] items-center': variant !== 'vertical',
+    'flex-col': variant === 'vertical',
+  });
+
+  const avatarVariant =
+    variant === 'big' || variant === 'regular' ? variant : undefined;
+
+  const contentClasses = cx('', {
```

Comment:
You can skip first argument if no constant class is needed 👍 

```suggestion
  const contentClasses = cx({
```